frontend/src/scenes/notebooks/NotebookCommentButton/NotebookCommentButton.tsx

```diff
@@ -0,0 +1,72 @@
+import { LemonButton, LemonButtonProps } from 'lib/lemon-ui/LemonButton'
+import { LemonMenu } from 'lib/lemon-ui/LemonMenu'
+import { IconComment } from 'lib/lemon-ui/icons'
+import { Spinner } from 'lib/lemon-ui/Spinner'
+import { notebookCommentButtonLogic } from 'scenes/notebooks/NotebookCommentButton/notebookCommentButtonLogic'
+import { useValues } from 'kea'
+import { LemonMenuProps } from 'lib/lemon-ui/LemonMenu/LemonMenu'
+
+interface NotebookCommentButtonProps extends Pick<LemonButtonProps, 'size'>, Pick<LemonMenuProps, 'visible'> {
+    sessionRecordingId: string
+    onCommentInNewNotebook: () => void
+    onCommentInExistingNotebook: (notebookShortId: string) => void
+}
+
+/** A wrapper for the button, that lets you open a new or an existing recording
+ */
+export function NotebookCommentButton({
+    sessionRecordingId,
+    onCommentInNewNotebook,
+    onCommentInExistingNotebook,
+    size = 'small',
+    // allows a caller to determine whether the pop-over starts open or closed
+    visible = undefined,
+}: NotebookCommentButtonProps): JSX.Element {
+    const logic = notebookCommentButtonLogic({ sessionRecordingId })
+    const { notebooksLoading, notebooks } = useValues(logic)
+    return (
+        <LemonMenu
+            visible={visible}
+            items={[
+                {
+                    items: notebooksLoading
+                        ? [
+                              {
+                                  sideIcon: <Spinner />,
+                                  active: true,
+                                  label: 'Finding notebooks with this recording...',
+                                  onClick: () => {
```

Comment:
I'm assuming this is a type check thing that the handler needs to be specified even if not used?

Reply:
Ah, it was also mainly a laziness thing 😊 

Will check if there's already a better way and use that or add it before merging 👍